.github/workflows/update-docs-in-website.yml

```diff
@@ -0,0 +1,50 @@
+name: Update latest CLI documentation in the website
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'docs/*.md'
+
+jobs:
+  Make-PR:
+    name: Make PR on website repository with updated latest CLI documentation
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - name: Checkout Current repository
+        uses: actions/checkout@v2
+        with:
+          path: cli
+      - name: Checkout Another repository
+        uses: actions/checkout@v2
+        with:
+          repository: asyncapi/website
+          path: website
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Config git
+        run: |
+          git config --global user.name asyncapi-bot
+          git config --global user.email info@asyncapi.io
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-cli-docs-${{ github.sha }}
+      - name: Copy cli folder from Current Repo to Another
+        working-directory: ./website
+        run: |
+          mkdir -p ./pages/docs/tools/cli
+          printf "%s\ntitle: cli\nweight: 4\n%s" "---" "---"> ../cli/docs/_section.md
```

Comment:
I made code suggestions to this line but they are not applied

Reply:
oh yes! I increase the weight from `3` to `4` how high would you suggest the weight to be?

Reply:
did you see https://github.com/asyncapi/cli/pull/476#discussion_r1143379055 comment?

Reply:
@derberg I've made pushed a new commit

Reply:
@activus-d I think you missed https://github.com/asyncapi/cli/pull/476#discussion_r1143379055 to make title uppeer case `CLI` and not `cli`

Reply:
@derberg I've pushed a new commit to implement the review